### DISCUSSION TITLE
How to get the slash command that generated a message

### DISCUSSION THREAD

**Discussion by kduytsch** [Rapptz/discord.py#9401] [Q&A]
I'm writing a helper bot to alert me for some specific games on discord. Those games are started by slash commands, and I can't get them via a standard on_message

Any hint how to achieve this ?

In the GUI, we see "xxx used /yyy", I was thinking a bit like for a reply to have a parent message or other, but can't find anything

Thank you

**Comment by LostLuma** [comment]
Hello, you can get information about the command used via [Message.interaction](https://discordpy.readthedocs.io/en/latest/api.html#discord.Message.interaction), including its full name.
